src/en/components/side-navigation/base.md

```diff
@@ -0,0 +1,26 @@
+---
+github: https://github.com/cds-snc/gcds-components/tree/main/src/components/gcds-side-navigation
+figma: https://www.figma.com/file/mh2maMG2NBtk41k1O1UGHV/Canadian-Digital-Service%E2%80%A8---GC-Design-System?type=design&node-id=5633-11428&mode=design&t=4ltBpy3FPMc9pXcL-0
+permalink: false
+tags: ['sidenavigationEN', 'header']
+---
+
+# Side navigation <br>`<gcds-side-nav>`
+
+_Also called: sidebar menu, lefhand nav, vertical navigation._
+
+A side navigation is a vertical list of page links on the left side of the screen.
+
+{% docLinks locale stage figma github %}
+{% enddocLinks %}
+
+{% componentPreview "Side navigation component preview" "pt-400 pb-200" %}
+<gcds-side-nav label="Side navigation component preview">
```

Comment:
Is it repetitive to use the same heading here for the preview container and the label? They get both displayed right below each other,

Reply:
I did think it looked kind of off but only the side-nav's label is a heading so I don't think it will cause any confusion.